### ISSUE TITLE
Midpoint Review - Accessibility Feedback - Accredited Representation Management, Appoint a Representative v2 + Find a Representative v3

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#83546]
## Next Steps for the VFS team

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions
- @mtri I really appreciate how game you were during my feedback on Friday! Thank you for pivoting so gracefully to the search results prototype.

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [x] **Should:** I think in this context the type-ahead search might just be too complex of an interaction for screen reader users to reliably complete. Each result holds multiple pieces of information, meaning a very high cognitive load for users that don't get the visual cues separating each result in the dropdown. As discussed in the meeting, I like the direction you explored where the interaction is broken up as a more *search* then *results* then *select from the results* flow.
   - [ ] **Alternative should:** If you decide that the type-ahead is desirable enough to pursue, some recommendations:
      - See how much you can simplify the results in the dropdown. Can you just give city/state instead of the full address? Can you remove "Accepts digital signature" and instead have a filter option before the search tool? etc.
      - Make sure that any type-ahead component matches the keyboard navigation and screen reader announcements in the design system search-with-type-ahead component. We want to make sure users get a consistent experience rather than having to learn different interaction patterns. If you go this route, definitely work with someone from CAIA and please loop me in too, and I can give you more detailed acceptance criteria.
- [x] **Engineering note:** When you've selected a representative, keyboard focus should move to somewhere using `tabindex="-1"` that gives users a clear update that the action has succeeded. Whatever gets keyboard focus will be immediately announced to screen reader users. It doesn't necessarily have to share everything, just enough to say *yes, the thing I just did worked.*
   - [ ] **Consider:** Moving focus to "You've selected this accredited representative:" is sufficient, but maybe tinker with the phrasing to see if you can make it stand alone a little more cleanly, eg. "You've selected an accredited representative."
- [x] **Should:** Echoing comments from others during the meeting, see if you can simplify the [card that's displayed once you've selected a representative](https://www.figma.com/proto/bzbwObT9hiItve0q3cQX9c/Find-and-Appoint-a-Representative?page-id=5138%3A420728&type=design&node-id=4521-226002&viewport=10254%2C-11262%2C0.5&t=xviodnSh9GXwnmFK-1&scaling=min-zoom&starting-point-node-id=3717%3A80893). There's a lot of information packed in there that's maybe not relevant at this stage of the flow, which increases cognitive load a bit (and makes the screen reader experience pretty verbose).

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [ ] Assign this ticket to the VFS team member that opened the Slack request
- [ ] Add the VFS team product label
- [ ] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [x] Add the Collaboration Cycle initiative milestone

**Comment by oddball-lindsay** [comment]
@briandeconinck does the Engineering note also apply to the design that has the [card that's displayed once you've selected a representative](https://www.figma.com/proto/bzbwObT9hiItve0q3cQX9c/Find-and-Appoint-a-Representative?page-id=5138%3A420728&type=design&node-id=4521-226002&viewport=10254%2C-11262%2C0.5&t=xviodnSh9GXwnmFK-1&scaling=min-zoom&starting-point-node-id=3717%3A80893)?

**Comment by briandeconinck** [comment]
@oddball-lindsay Thank you for the nudge in Slack, and sorry for the slow reply!

Yes, the same feedback applies: When you make a selection, keyboard focus should move to something telling you what you've selected.

**Comment by oddball-lindsay** [comment]
Great thanks, will make sure to include this in implementation details when we get to development :)
